README.md

```diff
@@ -182,14 +183,16 @@ The repo contains many packages, each which may have dependencies on each other.
 npm install -g @microsoft/rush
 ```
 
-To use rush to build, you can run `rush build`, which will incrementally build the entire repo (only build what has changed since the last build.)
+To use rush to build, you can run `rush build`, which will incrementally build the entire repo (only build what has changed since the last build.) If you don't have rush globally installed, you can also use `npm run buildfast` to achieve this, since this command abstracts `rush build`.
 
-To can also build up to a specific project using the `--to <package>` argument. For example, to build up to `office-ui-fabric-react`, you can run:
+You can also build up to a specific project using the `--to <package>` argument. For example, to build up to `office-ui-fabric-react`, you can run:
 
 ```bash
 rush build --to office-ui-fabric-react
 ```

```

Comment:
should probably mention buildfast here instead or as well.

Reply:
I do mention buildfast just before that paragraph :)